learn/what_is_meilisearch/features.md

```diff
@@ -18,51 +18,48 @@ Meilisearch's default [relevancy rules](/learn/core_concepts/relevancy.md) are d
+Create [filters](/learn/advanced/filtering_and_faceted_search.md) to refine results based on user-defined criteria.
 
 ## Faceting
 
-[Faceted search](/learn/advanced/filtering_and_faceted_search.md) allows you to classify search results into categories and to build intuitive navigation interfaces.
+Build intuitive navigation interfaces with [faceted search](/learn/advanced/filtering_and_faceted_search.md).
 
 ## Sorting
 
-[Sort search results](/learn/advanced/sorting.md) at query time and let users choose which types of results they want to see first.
-
-## Placeholder search
+[Sort search results](/learn/advanced/sorting.md) at query time and let users choose which results they want to see first.
 
-If you make a search without inputting any query words, Meilisearch will return all the documents in that index sorted by its [custom ranking rules](/learn/core_concepts/relevancy.md#custom-rules) and [sorting rules](/learn/advanced/sorting.md#sorting). This feature is called **placeholder search**.
+## API key management
 
-Placeholder searches are particularly effective when used with other features such as [faceting or filtering](/learn/advanced/filtering_and_faceted_search.md#filters-or-facets), which allow users to narrow their searches and browse by category. You can read more about this feature in our article on [search parameters](/reference/api/search.md#placeholder-search).
+[Protect your instance with API keys](/learn/security/master_api_keys.md). Control which users can access what indexes, routes, and endpoints.
 
-## Phrase search
+## Multitenancy
 
-If you enclose search terms in double quotes (`"`), Meilisearch will only return documents that contain those terms in the order they were given. This gives users the option to make more precise search queries.
+Manage complex multi-user applications handling sensitive data. [Tenant tokens](/learn/security/tenant_tokens.md) help you decide which documents each one of your users can search.
 
-Phrase search is particularly useful when looking for a single result, such as searching for a book by ISBN. It is also possible to combine phrase searches with Meilisearch's basic syntax so only parts of a query are matched strictly.
+## Synonyms
 
-## API key management
+[Define synonyms](/learn/configuration/synonyms.md) and craft a tailored, intuitive search experience.
 
-Meilisearch allows you to protect your instances through the use of API keys. API keys give you control over which users can access what indexes, routes, and endpoints.
+## Highlighting
 
-API keys are managed by the master key. When you set a master key on your first launch, Meilisearch generates two API keys called the `Default Search API Key` and the `Default Admin API Key`. You can read more them in our article on [security](/learn/security/master_api_keys.md).
+[Highlight](/reference/api/search.md#attributes-to-highlight) query terms and help users immediately find the matched text in a document.
```

Comment:
As I mentioned above, this whole page feels too subjective. I think it's fine, since we have "Ultra relevant" listed as a feature when it's arguably more of a promise, but I'm not sure the docs team is well-equipped to make this call.

Reply:
I think we should keep it for the sake of competitor parity, it may help to get a positive first impression and save the visitor from looking deeper just to check that.

As long as the features are well organized, I think there is no harm in being exhaustive about user-facing capacities.